### ISSUE TITLE
[FeatureRequest] Hide 'Clip' Button

### ISSUE THREAD

**Issue by roflsunriz** [lawrencehook/remove-youtube-suggestions#143]
![スクリーンショット 2024-10-08 103706](https://github.com/user-attachments/assets/7d2315c5-aeff-4738-9ed2-9b990e803745)
I want to hide this ✂️Clip Button.

**Comment by roflsunriz** [comment]
For reference, the CSS selector is usually 
```css
ytd-menu-renderer.ytd-watch-metadata > div:nth-child(2) > yt-button-view-model:nth-child(2) > button-view-model:nth-child(1) > button:nth-child(1) > yt-touch-feedback-shape:nth-child(3) > div:nth-child(1) > div:nth-child(1)
```

But I tried to not to display this button via uBlockOrigin in 'my filter' section, like 
```
www.youtube.com##ytd-menu-renderer.ytd-watch-metadata > div:nth-child(2) > yt-button-view-model:nth-child(2) > button-view-model:nth-child(1) > button:nth-child(1) > yt-touch-feedback-shape:nth-child(3) > div:nth-child(1) > div:nth-child(1)
```
 but it doesn't work.

**Comment by lawrencehook** [comment]
bb5f4331eff2b47612f61538be9de710cef8f881

release pending!

**Comment by lawrencehook** [comment]
released as of version `4.3.62`
